doc/authors.md

```diff
@@ -1,36 +1,45 @@
 Zcash Contributors
 ==================
 
-Jack Grigg (323)
-Simon Liu (220)
-Sean Bowe (180)
+isle2983 (2983)
```

Comment:
Ah, this regex is buggy (parens not escaped): https://github.com/zcash/zcash/blob/master/zcutil/release-notes.py#L20